style.css

```diff
@@ -291,3 +312,79 @@ h4 {
 #contacts-section {
   display: none;
 }
+
+#name {
+  color: #6b778c;
+  font-size: 15px;
+  line-height: 24px;
+  width: 295px;
+  height: 48px;
+  text-align: left;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+  padding: 10px;
+}
+
+#email {
+  color: #6b778c;
+  font-size: 15px;
+  line-height: 24px;
+  width: 295px;
+  height: 48px;
+  text-align: left;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+  padding: 10px;
+}
+
+#message {
+  font-weight: 900;
+  line-height: 24px;
+  width: 303px;
+  height: 114px;
+  text-align: left;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+  padding: 10px;
+}
+
+.inputs {
+  display: grid;
+  margin: auto;
+  width: 327px;
+  height: 330px;
+  text-align: center;
+}
+
+.submit {
+  height: 48px;
+  width: 129px;
+  border-radius: 4px;
+  background-color: #36b37f;
+  color: #fff;
+  display: flex;
+  align-items: center;
+  margin: auto;
+  border: 1px solid #36b37f;
+  justify-content: center;
+}
```

Comment:
I apologize if no reviewer mentioned this before, but none of your buttons have any interaction stylings. This has been a project requirement all along. Please implement the stylings for all of your buttons in your entire project, I'm marking just this one for brevity.

<img width="762" alt="Screen Shot 2022-06-08 at 14 58 54" src="https://user-images.githubusercontent.com/82426682/172706057-a920021e-18f9-494c-a1d9-4e1a5975ca80.png">

<img width="904" alt="Screen Shot 2022-06-08 at 15 00 27" src="https://user-images.githubusercontent.com/82426682/172706287-6588ff3a-4c33-47d6-a7cb-8ef645451b53.png">